packages/@guardian/source-react-components-development-kitchen/src/toggle-switch/styles.ts

```diff
@@ -110,6 +111,11 @@ export const toggleStyles = (format?: ArticleFormat): SerializedStyles => {
 			left: 22px;
 			background: ${neutral[100]};
 		}
+
+		&:focus-visible {
+			outline: 5px solid ${format ? neutral[100] : brand[500]};
+			outline-offset: 3px;
```

Comment:
is there any chance this could overlap existing elements? if so, this might be considered a breaking change (consumers could need to account for some extra border)

or is the space it consumes already accounted for elsewhere?

Reply:
<img width="416" alt="Screenshot 2022-10-11 at 18 03 29" src="https://user-images.githubusercontent.com/77005274/195155406-376a63fe-ded7-4072-a62e-7494f4682a24.png">

Reply:
The ToggleSwitch component itself doesn't contain any padding but where it's consumed here in DCR liveblogs it fits ok within the space. I'm not sure what that means in terms of whether its a breaking change though?